packages/react-radio/src/components/RadioGroup/useRadioGroupStyles.ts

```diff
@@ -1,7 +1,14 @@
 import { makeStyles, mergeClasses } from '@griffel/react';
-import { RadioGroupState } from './RadioGroup.types';
+import { RadioGroupSlots, RadioGroupState } from './RadioGroup.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
 
+/**
+ * @deprecated
+ */
 export const radioGroupClassName = 'fui-RadioGroup';
```

Comment:
For `RadioGroup` you can actually remove this class name instead of deprecating it since the component is still under `unstable`:

```suggestion
```